style/style.css

```diff
@@ -0,0 +1,518 @@
+* {
+  font-family: "Delius Unicase", cursive;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  background: #ffffff;
+  user-select: none;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 120px auto 120px auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: fit-content fit-content fit-content;
+  gap: 120px 0px;
+  grid-template-areas:
+    "cities"
+    "nature"
+    "people";
+}
+
+div[class^="image"] {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover, cover;
+  border-radius: 20px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  padding: 20px;
+  grid-template-areas:
+    "icon-left icon-right"
+    "text-left text-right";
+}
+
+div[class^="image"]:hover {
```

Comment:
I see no need to specify the div tag here, attribute selector can be used on its own

Reply:
understand